### PR TITLE
Webbrowser: update to new module structure

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: modules/webbrowser/CMakeLists.txt [hunk 1]
```diff
@@ -27,9 +24,11 @@ DownloadCEF("${CEF_PLATFORM}" "${CEF_VERSION}" "${PROJECT_BINARY_DIR}/ext/cef")
 # Add the CEF binary distribution's cmake/ directory to the module path.
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CEF_ROOT}/cmake")
 
-#
+# sandbox lib is built with MT_StaticRelease but Inviwo is built with
```
Comment (ResearchDaniel) [inline modules/webbrowser/CMakeLists.txt:27]: Only on windows as far as I can recall.

File: modules/webbrowser/CMakeLists.txt [hunk 2]
```diff
@@ -65,16 +57,12 @@ if(CMAKE_COMPILER_IS_GNUCXX OR "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
 endif()
 # Logical target used to link the libcef library.
 ADD_LOGICAL_TARGET("libcef_lib" "${CEF_LIB_DEBUG}" "${CEF_LIB_RELEASE}")
-
 set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_TMP})
 
 # Determine the target output directory.
 # Inviwo: use module target directory as output dir$<TARGET_FILE_DIR:inviwo-module-webbrowser>
-#SET_CEF_TARGET_OUT_DIR()
```
Comment (ResearchDaniel) [inline modules/webbrowser/CMakeLists.txt]: I want to keep this for reference to the example cmake file provided by CEF.

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Minor changes requested. Would like to try it on my mac as well.

**Comment by petersteneteg** [comment]
would be great if you did try it on mac...
I think/hope that I didn't change any behaviour, but it is hard to know without testing...

File: modules/webbrowser/CMakeLists.txt (hunk 1 above)
Comment (petersteneteg) [reply modules/webbrowser/CMakeLists.txt:27]: ahh

File: modules/webbrowser/CMakeLists.txt (hunk 2 above)
Comment (petersteneteg) [reply modules/webbrowser/CMakeLists.txt]: ok

**Comment by ResearchDaniel** [comment]
Webhelper needs to depend on warn.
@Alboin has a fix

File: modules/webbrowser/webhelper/CMakeLists.txt [hunk 3]
```diff
@@ -47,6 +47,7 @@ target_link_libraries(${CEF_HELPER_TARGET} PRIVATE
 	$<$<NOT:$<BOOL:${OS_MACOSX}>>:libcef_lib>
 	libcef_dll_wrapper 
 	${CEF_STANDARD_LIBS}
+    inviwo::warn
```
Comment (ResearchDaniel) [inline modules/webbrowser/webhelper/CMakeLists.txt]: Already added on line 32. Remove line 32 perhaps?

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]

**Review by ResearchDaniel — APPROVED** [review]

**Pull request merged**